test/dlc_tests/container_tests/bin/benchmark/run_pytorch_training_performance_gpu_imagenet

```diff
@@ -49,7 +49,7 @@ echo Preparation Stage took $(( DIFF/60 )) mins to finish>&2
 
 echo Benchmark started: >&2
 START=$(date +%s)
-trigger_test "100m" "run" 2>&1 | tee ${LOG_DIR}/${LOG_FILE}
+trigger_test "110m" "run" 2>&1 | tee ${LOG_DIR}/${LOG_FILE}
```

Comment:
Tested on p3.16xlarge for a 100m run. Epoch 5 was not complete. Adjust it to 110m